policy/modules/system/systemd.te

```diff
@@ -751,6 +754,11 @@ allow systemd_networkd_t self:tun_socket { create_socket_perms relabelfrom relab
 allow systemd_networkd_t self:udp_socket create_socket_perms;
 allow systemd_networkd_t self:unix_dgram_socket create_socket_perms;
 
+allow systemd_networkd_t system_dbusd_runtime_t:dir read;
+allow systemd_networkd_t system_dbusd_runtime_t:sock_file read;
+allow systemd_networkd_t var_run_t:dir read;
+allow systemd_networkd_t fs_t:filesystem getattr;
```

Comment:
`fs_getattr_xattr_fs(systemd_networkd_t)`

Reply:
I'll use this interface, thanks!